tests/test_ec2deprecateimg.py

```diff
@@ -21,89 +21,415 @@
 
 import logging
 import pytest
-import datetime
-import dateutil.relativedelta
+import os
+import argparse
 
-import ec2imgutils.ec2deprecateimg as ec2depimg
+# Hack to get the script without the .py imported for testing
```

Comment:
The recommended approach to achieve cross-platform compatibility is to use console_scripts entry points instead of scripts (in setup.py). This would solve the problem of discoverability although it would be a bigger change.